reflectance_api.py

```diff
@@ -8,11 +9,10 @@ def getReflectanceListbyPoint1992(point):
     x = point.x()
     y = point.y()
 
-    URL = "https://mapy.geoportal.gov.pl/wss/service/PZGIK/OI/WMS/SkorowidzeObrazowIntensywnosci?"
     LAYERS = [
-            'SkorowidzeOI',
-            'SkorowidzeOIZasieg'
-        ]
+        'SkorowidzeOI',
+        'SkorowidzeOIZasieg'
+    ]
```

Comment:
może to też wrzucisz do constants. Byłby słownik z warstwami dla okreslonych produktów z którego byłyby wydobywane takie warstwy